R/fda-table_7.R

```diff
@@ -0,0 +1,87 @@
+library(dplyr)
+library(tern)
+
+# Load and pre-process data
+
+adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl")
+ntrtem <- sample(adsl[["USUBJID"]], size = as.integer(nrow(adsl) / 3)) # assign subjects to nontreatment emergent AE's
+
+adae <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adae") %>%
+  filter(SAFFL == "Y", DTHFL == "Y") %>% # not actually necessary because all observations are flagged as Y
+  mutate(
+    TRTEMFL_ = ifelse(USUBJID %in% ntrtem, "N", "Y"), # produce dummy data
+    trtemlab = case_when( # produce label column
+      TRTEMFL_ == "Y" ~ "Treatment-emergent deaths (3)",
+      TRTEMFL_ == "N" ~ "Nontreatment-emergent deaths (4)",
+      .default = "Other"
+    )
+  )
+
+
+# Set layout
+# Note: Does not count re-animated patients who "died twice"
+
+lyt <- basic_table(show_colcounts = TRUE) %>%
+  split_cols_by("ARM") %>%
+  analyze_num_patients(
+    var = "USUBJID",
+    .stats = "unique",
+    .labels = c(unique = "Total deaths"),
+    denom = "N_col",
+    indent_mod = -1L,
```

Comment:
Since the "Deaths" label is already in the leftmost position, this has no effect and can be removed. 
```suggestion
```